docs/RC/GetInteriorVehicleData/index.md

```diff
@@ -0,0 +1,103 @@
+## GetInteriorVehicleData
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: To read RC module status data. The same function is used to subscribe/unsubscribe on RC module status/setting change notifications.
+
+### Request
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|moduleType|[Common.ModuleType](https://github.com/OlesiaV/sdl_hmi_integration_guidelines/blob/sdl_rc_baseline/docs/Common/Enums/index.md)|true||
+|subscribe|Boolean|false|defvalue="false"|
+|appID|Integer|true||
+
+### Response
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|moduleData|[Common.ModuleData](https://github.com/OlesiaV/https://github.com/OlesiaV/sdl_hmi_integration_guidelines/blob/master/docs/Common/Structs/index.md)|true||
+|isSubscribed|Boolean|false||
+
+### Sequence Diagrams
+|||
```

Comment:
This chart is not displaying properly. I think there needs to be a line between lines 31 and 32

Reply:
@JackLivio fixed